### PR TITLE
Only set changed_saved_companies cookie if the user actually changed something

### LINKED ISSUE #433: Only set changed_saved_companies cookie if the user actually changed something
In the wizard on the homepage, we use the `changed_saved_companies` cookie to determine whether the user has changed the default suggested companies (to test, use Germany, Austria or the UK as the country; we only have suggested companies for them). However, this cookie is unnecessarily set whenever the user enters the homepage.

Instead, we should set the cookie only if the user actually changed something. The cookie should not be set if the user just loads the homepage. In addition, it should instead be deleted whenever we currently set it to `false`.

The relevant code is in this file: https://github.com/datenanfragen/website/blob/master/src/Components/Wizard.js

### PR DESCRIPTION
**Pull request by alejandrotoga02**
# Only set changed_saved_companies cookie if the user actually changed something

fix #433

### CODE REVIEW

File: src/Components/Wizard.js [hunk 1]
```diff
@@ -238,10 +238,14 @@ export class SavedCompanies {
     }
 
     getUserChanged() {
-        return Cookie.get(USER_CHANGED_COOKIE) === 'true';
+        return Cookie.get(USER_CHANGED_COOKIE) ? true : false;
```
Comment (baltpeter) [inline src/Components/Wizard.js:241]: You should be able to just convert to a boolean like this:

```suggestion
        return !!Cookie.get(USER_CHANGED_COOKIE);
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the changes, @alejandrotoga02! I have just left one small comment below.

Also, please make sure that you run Prettier with our configuration of the file to make sure it matches our formatting style. If you follow the instructions in the README, this should happen automatically on committing.

**Comment by alejandrotoga02** [comment]
@baltpeter  okok  I'm going to do the changes :D

File: src/Components/Wizard.js (hunk 1 above)
Comment (alejandrotoga02) [reply src/Components/Wizard.js]: done

**Review by baltpeter — APPROVED** [review]
Great, thanks again.

**Pull request merged**